### DISCUSSION TITLE
Input Textbox for passwords

### DISCUSSION THREAD

**Discussion by tilusnet** [OliveTin/OliveTin#339] [Q&A]
Hi,

I find this project very useful, thanks for it.

I was wondering, would it be easy to add a "password" style input textbox where characters are shown as ******** during input?

**Comment by jamesread** [comment]
Yep, I just added password support, it will be in the next release.

**Reply by tilusnet** [reply]
Fantastic, thank you!

**Comment by jamesread** [comment]
Documented here now by the way; https://docs.olivetin.app/args/password.html
